CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Unreleased (6 March 2017)
+
+- Search and replace `Grid` with `List`.
```

Comment:
This needs to be better. There should be a sub-heading `### BREAKING CHANGES` for documenting changes that a user MUST make if upgrading to this version. There should be a sub-heading `### IMPROVEMENTS` to document the improvements that are in the next release. One feature might appear in both categories. Use the previous entries as a guide.